addons/account/models/account_account.py

```diff
@@ -779,9 +767,9 @@ class AccountGroup(models.Model):
 
     parent_id = fields.Many2one('account.group', index=True, ondelete='cascade', readonly=True)
     parent_path = fields.Char(index=True, unaccent=False)
-    name = fields.Char(required=True)
-    code_prefix_start = fields.Char()
-    code_prefix_end = fields.Char()
+    name = fields.Char(required=True, translate=True)
+    code_prefix_start = fields.Char(compute='_compute_code_prefix_start', readonly=False, store=True, precompute=True)
+    code_prefix_end = fields.Char(compute='_compute_code_prefix_end', readonly=False, store=True, precompute=True)
```

Comment:
To avoid an `UPDATE` query for each row later one. This way, everything is done in the `INSERT` query.
Basically, `precompute=True` should be the default but it isn't the case "to be safe" as it can theoretically break some code in some cases